server/events/vcs/github_client.go

```diff
@@ -330,7 +330,7 @@ func (g *GithubClient) PullIsApproved(repo models.Repo, pull models.PullRequest)
 				return models.ApprovalStatus{
 					IsApproved: true,
 					ApprovedBy: *review.User.Login,
-					Date:       *review.SubmittedAt,
+					Date:       review.SubmittedAt.Time,
```

Comment:
`review.SubmittedAt` is always existed, cause `if review != nil && review.GetState() == "APPROVED"` is already checked before.